app/views/projects/show.html.erb

```diff
@@ -1,33 +1,86 @@
-<div class="container mx-auto">
-  <div class="mt-5">
-    <div class="mb-3">
-      <%= @project.title %>
+<div class="page-wrapper">
+  <div class="page-header d-print-none text-white">
+    <div class="container-xl">
+      <div class="row g-2 align-items-center">
+        <div class="col">
+          <div class="page-pretitle">
+            Project overview
+          </div>
+          <h2 class="page-title">
+            <%= link_to @project.title, project_path(@project) %>
+          </h2>
+        </div>
+        <div class="col-auto ms-auto d-print-none">
+          <div class="btn-list">
+            <span class="dropdown">
+              <a href="#" class="btn btn-primary dropdown-toggle align-text-top show" data-bs-boundary="viewport" data-bs-toggle="dropdown">
+                <%= inline_svg_pack_tag('static/@tabler/icons/icons/plus.svg') %>
+                New
+              </a>
+              <div class="dropdown-menu dropdown-menu-end" data-popper-placement="bottom-end" style="position: absolute; inset: 0px 0px auto auto; margin: 0px; transform: translate3d(0px, 29.5px, 0px);">
+                <%= link_to 'Account', new_project_account_path(@project) %>
+              </div>
+            </span>
+            <% if policy(@project).edit? %>
+              <a class="btn btn-primary" href="#">
```

Comment:
Nem mutat sehova a link, pedig van már projekt szerkesztés oldal

Reply:
Ha ez mozgott át a `/projects/_header.html.erb:29` -be, akkor még nem mutat sehova a link.